cfgov/cfgov/settings/base.py

```diff
@@ -103,6 +103,18 @@
         {'import': 'eregs', 'apps': ('eregs_core',)},
     ]
 
+# These apps use "legacy" database routing
+# See: cfgov/v1/db_router.py
+LEGACY_APPS = [
+    'comparisontool',
+    'paying_for_college',
+    'retirement_api',
+    'knowledgebase',
```

Comment:
Should this be removed from the list, now that `knowledgebase` is no longer an optional app?